File: Command/Golang/Editor.go
package main

type Editor struct {
	Commands []ICommand
}

func (editor *Editor) input(command ICommand) {
	editor.Commands = append(editor.Commands, command)
}

func (editor *Editor) executeCommands() {
	for _, command := range editor.Commands {
		command.execute()
	}

	editor.Commands = []ICommand{}
}

File: Command/Dart/Editor.dart
import 'ICommand.dart';

class Editor {
  List<ICommand> _commands = [];

  void input(ICommand command) {
    _commands.add(command);
  }

  void executeCommands() {
    for (var command in _commands) {
      command.execute();
    }

    _commands.clear();
  }
}